code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -171,6 +171,11 @@
 	turns_since_move = turns_per_move
 	..()
 
+/mob/living/simple_animal/revive(reset_to_roundstart)
+	. = ..()
+	blood_amount = initial(blood_amount)
+	bleeding = FALSE
```

Comment:
Is this properly called when rejuvenate is called on a living mob, and does this clear the blood overlays?

Reply:
yes